src/GraphControl/Control/Grapher.cpp

```diff
@@ -894,3 +858,27 @@ String ^ Grapher::ConvertToLinear(String ^ mmlString)
 
     return ref new String(linearExpression.c_str());
 }
+
+void Grapher::OnAxesColorPropertyChanged(Windows::UI::Color /*oldValue*/, Windows::UI::Color newValue)
+{
+    if (m_graph)
+    {
+        auto axesColor = Graphing::Color(newValue.R, newValue.G, newValue.B, newValue.A);
+        m_graph->GetOptions().SetAxisColor(axesColor);
```

Comment:
I am surprised these don't get reset when we reinitialize m_graph, normally we would want to add calls to these in UpdateGraphOptions() also.